Comp2024/src/main/java/frc/robot/Robot.java

```diff
@@ -168,7 +168,10 @@ public void autonomousInit( )
    */
   @Override
   public void autonomousPeriodic( )
-  {}
+  {

```

Comment:
Ok for testing, but we can't keep this in the code due to the excessive spamming. Also, since this is instrumentation for the drivetrain, it really should be in the periodic task for the CommandSwerveDrivetrain. We can add an @Override void periodic () method to that class and use it. Also, I think the CTRE logger should track these values, so we shouldn't need the DataLogManager in the long term.

Reply:
Ok for merge, but fix on next pass. And investigate the CTRE logging capability.